### ISSUE TITLE
Cookie Banner 'hide this message' button unclear when navigating out of context

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk-design-system#2087]
## Description of the issue

Reported by users via Slack and the [Design System backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/12#issuecomment-1055590778). This was flagged to them in an accessibility audit by DAC.

## Steps to reproduce the issue

1. Open the [cookie banner confirmation message](https://design-system.service.gov.uk/components/cookie-banner/server-side-confirmation/index.html)
2. Tab to the button with a screenreader. The button is announced as 'Hide this message, button'. 
3. Can also be reproduced by navigating via form elements, e.g: in Mac Voiceover press VO+U and use the left and right arrows to navigate to the 'form elements' list. You should see 'Hide this message, button' listed.

## Actual vs expected behaviour

The cookie banner 'hide this message button' doesn't have enough information when read out of context. It's not clear what the button relates to unless you already know you're inside the cookie banner.

The text is something which is passed into the component so can be changed by services themselves. Suggestion to change the examples in the design system (and in our own review-app in govuk-frontend) to something that makes sense out of context, for example: "hide cookie banner".

## Who needs to work on this
Content Design; Developer

## Done When

- [x] Agreed new content for the button
- [ ] Updated examples in the design system
- [ ] Updated the design system website cookie banner
- [ ] Updated examples in the govuk-frontend review app
- [ ] Updated any guidance in the design system (where necessary)
- [ ] Draft comms for backlog and Slack/support queries

**Comment by lfdebrux** [comment]
We should update the [community backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/12) when we start work on this, as we're being chased about it on Slack.
